ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -232,6 +232,7 @@
       owner: root
       group: root
       mode: 0644
+    become: 'true'
```

Comment:
Looks like 2 "become: 'true'" for this 1 task...
Maybe the become needs to be before register, unsure..

@fabianvf any thoughts?